lib/vector-core/src/fanout.rs

```diff
@@ -130,11 +130,30 @@ impl Fanout {
         }
     }
 
-    pub async fn send_stream(&mut self, events: impl Stream<Item = EventArray>) {
+    /// Send a stream of events to all connected sinks.
+    ///
+    /// This function will send events until the provided stream finishes. It will also block on the
+    /// resolution of any pending reload before proceeding with a send operation, similar to `send`.
+    ///
+    /// # Panics
+    ///
+    /// This method can panic if the fanout receives a control message that violates some invariant
+    /// about its current state (e.g. remove a non-existent sink, etc). This would imply a bug in
```

Comment:
```suggestion
    /// about its current state (e.g. remove a non-existent sink, etc.). This would imply a bug in
```

Technically? IDK english is hard.